yellow-paper/docs/data-publication-and-availability/index.md

```diff
@@ -115,11 +145,14 @@ Briefly the concerns we must have for any supported data layer that is outside t
 
 #### Celestia
 
-Celestia mainnet is starting with a limit of 2 mb/block with 12 second blocks supporting ~166 KB/s. But they are working on increasing this to 8 mb/block.
+Celestia mainnet is starting with a limit of 2 mb/block with 12 second blocks supporting ~166 KB/s.
+But they are working on increasing this to 8 mb/block.
 
-As Celestia has just recently launched, it is unclear how much competition there will be for the data throughput, and thereby how much we could expect to get a hold of. Since the security assumptions differ greatly from the host chain (Ethereum) few L2's have been built on top of it yet, and the demand is to be gauged in the future.
+As Celestia has just recently launched, it is unclear how much competition there will be for the data throughput, and thereby how much we could expect to get a hold of.
+Since the security assumptions differ greatly from the host chain (Ethereum) few L2's have been built on top of it yet, and the demand is to be gauged in the future.
```

Comment:
```suggestion
Since the security assumptions differ greatly from the host chain (Ethereum) few L2s have been built on top of it yet, and the demand is to be gauged in the future.
```